tasks/main.yml

```
@@ -21,6 +21,9 @@
     name: "{{ service_name }}"
 
 - name: Configure chrony to use Amazon Time Sync Service
+  # AmazonLinux is already configured to use Amazon Time Sync Service
```

Minor nitpick, but might as well spell it consistently:
```suggestion
  # Amazon Linux is already configured to use Amazon Time Sync Service
```